src/GraphQL.Tests/Execution/InputConversionNewtonsoftJsonTests.cs

```diff
@@ -1,6 +1,9 @@
 namespace GraphQL.Tests.Execution;
 
+[Collection("StaticTests")]
 public class InputConversionNewtonsoftJsonTests : InputConversionTestsBase
 {
-    protected override Inputs VariablesToInputs(string variables) => variables.ToInputs();
+    private static readonly IGraphQLTextSerializer _serializer = new NewtonsoftJson.GraphQLSerializer(indent: true);
+    protected override Inputs VariablesToInputs(string variables)
+        => _serializer.Deserialize<Inputs>(variables) ?? Inputs.Empty;
```

Comment:
Maybe do the same in STJ tests for consistency and to make the use of required serializer explicit?

Reply:
Good idea

Reply:
Done